lua/nvim_lsp/intelephense.lua

```diff
@@ -11,22 +11,38 @@ local installer = util.npm_installer {
   binaries = {bin_name};
 }
 
+local function is_inside_cwd(path, cwd)
```

Comment:
I would call this `util.path.is_descendant(root, path)`, it doesn't really have anything to do with CWD.

OTOH we should probably mimic Python's `commonpath()` function. https://stackoverflow.com/a/37923477/152142

Reply:
Yeah you're right. In the context I was thinking it was about CWD but you could pass any path.

Reply:
I have refactored/extracted `is_inside_cwd` into `util.path.is_descendant`.